m46x/StdDriver/src/fmc.c

```diff
@@ -202,7 +202,7 @@ int32_t FMC_Erase(uint32_t u32PageAddr)
   * @note     Global error code g_FMC_i32ErrCode
   *           -1  Erase failed or erase time-out
   */
-int32_t FMC_EraseBank(uint32_t u32BankAddr)
+int32_t FMC_Erase_Bank(uint32_t u32BankAddr)
```

Comment:
why it is not part of StdDriver commit !

Reply:
```
$git log ./m46x/StdDriver/src/fmc.c
Author: cyliangtw <cyliang@nuvoton.com>
Date:   Fri Dec 23 17:37:04 2022 +0800
    Sync M467 STD drivers 2022/12/22
commit 6461402934142dd8ef2b7c91c9a69405f851adff
Author: cyliangtw <cyliang@nuvoton.com>
Date:   Tue Sep 13 19:34:35 2022 +0800
    NuMaker M46x support flash controller FMC driver
```
In the previous commit: 
I tried to follow our Nuvoton FMC API interface
 of other series [M480 FMC API](https://github.com/OpenNuvoton/M480BSP/blob/master/Library/StdDriver/src/fmc.c#L259), so I modified it at that time.

However, after dicussion with our M460 SDK owner, not change it in near future, so my last commit is fullly same with M460 StdDriver on 2022/12/23.

Reply:
Rebase & fixed in commit a99da0a4ec3cef717e5c3c2b5dbfa7747fa7ddb4